docs/frontend-examples/PASSWORD_RESET_INTEGRATION.md

```
@@ -0,0 +1,564 @@
+}
+```
+
+### Response (429 Too Many Requests)
+```json
+{
+  "detail": "Rate limit exceeded. Try again in 60 minutes."
+}
+```
+
+### Email de Confirmación (Usuario recibe)
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)